float32[14] delta_joint
